proto/buf.lock

```diff
@@ -4,21 +4,12 @@ deps:
   - remote: buf.build
     owner: cosmos
     repository: cosmos-proto
-    branch: main
     commit: 1935555c206d4afb9e94615dfd0fad31
-    digest: b1-TNqW6xj2Pjha5Uoj9a-5uOeRo4mwswKfyqMcN3I_gZ0=
-    create_time: 2021-12-02T22:04:00.31049Z
   - remote: buf.build
     owner: cosmos
     repository: gogo-proto
-    branch: main
-    commit: bee5511075b7499da6178d9e4aaa628b
-    digest: b1-rrBIustouD-S80cVoZ_rM0qJsmei9AgbXy9GPQu6vxg=
-    create_time: 2021-12-02T20:01:17.069307Z
+    commit: 6652e3443c3b4504bb3bf82e73a7e409
```

Comment:
```suggestion
    commit: bee5511075b7499da6178d9e4aaa628b
```

We should not use the latest just yet. The migration will be done in https://github.com/cosmos/cosmos-sdk/pull/13070

Reply:
Suggested version seems to work, applying